docs/en/ufs/CephFS.md

```diff
@@ -17,9 +17,7 @@ two different implementations of under storage system for [CephFS](https://docs.
 ## Prerequisites
 
 ### Deploy Alluxio binary package
-The Alluxio binaries must be on your machine. You can either
-[compile Alluxio]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}), or
```

Comment:
why is this link invalid?
isn't it https://docs.alluxio.io/os/user/stable/en/contributor/Building-Alluxio-From-Source.html ?

Reply:
fix was for EE, not OS. making the change there instead, sorry about that.